test/spacerequest/spacerequest.go

```diff
@@ -61,3 +63,21 @@ func WithStatusNamespaceAccess(namespaceAccess toolchainv1alpha1.NamespaceAccess
 		spaceRequest.Status.NamespaceAccess = []toolchainv1alpha1.NamespaceAccess{namespaceAccess}
 	}
 }
+
+func NewNamespace(spacename string) *corev1.Namespace {
+	labels := map[string]string{
+		toolchainv1alpha1.TypeLabelKey:     "tenant",
+		toolchainv1alpha1.ProviderLabelKey: toolchainv1alpha1.ProviderLabelValue,
+	}
+	if spacename != "nospace" {
```

Comment:
What happens if `== nospace` ? Is it still a valid namespace? Do think this ever returns an invalid namespace? I mean should we return `(*corev1.Namespace, error)` or not necessary?

Reply:
yes it makes sense. Sorry I missed the `test` part ... sigh...

Reply:
No worries! My pleasure getting more feedback/reviews and discussing alternative solutions! Thanks for checking!